advanced/android_gradle_plugin/README.md

```diff
@@ -0,0 +1,72 @@
+<!--
+SPDX-License-Identifier: MPL-2.0
+
+This file is part of Ramses Composer
+(see https://github.com/bmwcarit/ramses-composer-docs).
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+# Automatic binary generation
+
+This tutorial explains how to improve your workflow when designing Android apps with assets produced
+in the Ramses Composer.
+
+This tutorial extends the [basic Android tutorial](../../basics/android_app/README.md).
+
+## Typical artist workflow
+
+The Ramses toolchain is primarily aimed at a subgroup of graphics developers called `technical artists`. The typical workflow of a technical artist is similar to that of a full-stack developer. Changes and improvements are typically done iteratively, at small increments, and vertical in nature.
+
+In order to provide the most optimal workflow for technical artists, the Ramses toolchain provides
+a special Gradle plugin for Android. The plugin connects the development/application workflow with the
+design/composition workflow. Whenever you do a change in the original Ramses Composer scene, the plugin
+will pick it up, rebuild your app and integrate the latest/current version of the asset there.
+
+## Add the plugin
+
+You can add enable the plugin in your top-level build file using standard Gradle syntax (Gradle 7.2):
+
+```groovy
+// <app>/build.gradle
+plugins {
+    id 'io.github.bmwcarit.RaCoPlugin' version '0.3.1' apply false
+}
+
+```
+
+You can now enable it in all subprojects which have Ramses assets:
+
+
+```groovy
+// E.g. <app>/app/build.gradle
```

Comment:
this path looks fishy. what is <app> standing for? the app-module directory? and what is the second "app"
also in the path above before "plugins" statement

Reply:
Should I replace <app> with <root>? It stands for the root path where the example source code of the app lies inside this repository... Admittedly a bit confusing, got better idea?